### ISSUE TITLE
Error on building the project: `google-services.json` not found

### ISSUE THREAD

**Issue by noncom** [tranleduy2000/pascalnide#18]
I have downloaded the repo and am getting the following error on building the project:

    Error:Execution failed for task ':app:processDevDebugGoogleServices'.
    > File google-services.json is missing. The Google Services Plugin cannot function without it. 
       Searched Location: 
      /home/noncom/work/src/pascalnide/app/src/dev/debug/google-services.json
      /home/noncom/work/src/pascalnide/app/src/debug/dev/google-services.json
      /home/noncom/work/src/pascalnide/app/src/dev/google-services.json
      /home/noncom/work/src/pascalnide/app/src/debug/google-services.json
      /home/noncom/work/src/pascalnide/app/google-services.json

What is the recommended way to fix it?

**Comment by tranleduy2000** [comment]
You can create firebase project and download config file.
https://support.google.com/firebase/answer/7015592?hl=en

**Comment by noncom** [comment]
Eh, I did not yet use Firebase, so I don't have any experience with it. Okay, I have created a Firebase project, but there is not "gear" icon in it, that is mentioned in point 2 in the guide that you have referenced. There is, however "Project Settings" if I click the three vertical dots icon on the top right, but there's no "Your apps" card or any other card. Either I am missing something or that's Google changing their UIs too fast again.

In some other place, googling for info on `google.services.json`, I have found a link to this page: https://developers.google.com/mobile/add?platform=android&cntapi=analytics and I can specify my project there, but it also requires a package name. Do I have to specify any particular package name?
